### ISSUE TITLE
Angular Card Component

### ISSUE THREAD

**Issue by joaorbp** [dxc-technology/halstack-angular#315]
**Describe the bug**
Don't think the card component is styled following the guidelines. 

- The font family seems to be Roboto.
- The padding for the content of the card body seems to be incorrect.
- The margin of the card is incorrect.

**To Reproduce**
I'm just using the [developer site](https://developer.dxc.com/tools/angular/3/#/components/card), and the [design site](https://developer.dxc.com/design/components/card) for the guidelines.

**Expected behavior**

- Expected the font family to be Open Sans
- Expected the padding for the content of the card to be 20px top, 20px right, 20px bottom and 24px left
- Expected the margin to be 20px instead of 24px.

**Screenshots**
### Font family
![image](https://user-images.githubusercontent.com/1644143/114381155-b6ceec00-9b82-11eb-8ffe-04e540ae0dbf.png)

### Padding
![image](https://user-images.githubusercontent.com/1644143/114381324-e7168a80-9b82-11eb-924f-e8ef5f08587a.png)

### Margin
![image](https://user-images.githubusercontent.com/1644143/114381456-0dd4c100-9b83-11eb-9be4-d7565082d9f7.png)

**Additional context**
Based on the guidelines documentation:

### Font family
![image](https://user-images.githubusercontent.com/1644143/114381591-3361ca80-9b83-11eb-92b1-f2088eb2858a.png)

### Padding
![image](https://user-images.githubusercontent.com/1644143/114381665-470d3100-9b83-11eb-8e33-5fa66517e90d.png)

I also think in relation to the padding there is inconsistence between the text version and the image.

As in the image is 24px in the left, but the text says 20px in general.

![image](https://user-images.githubusercontent.com/1644143/114382767-ac155680-9b84-11eb-9147-96ad8b4a2ccb.png)

### Margin
![image](https://user-images.githubusercontent.com/1644143/114381714-57251080-9b83-11eb-948a-99bdf169d9b1.png)

### Also in documentation

The min height in documentation is 220px, but the image example is using 154px.

![image](https://user-images.githubusercontent.com/1644143/114388304-690ab180-9b8b-11eb-8c72-cb5b5cddbc8b.png)

The image max width is 140px but in the image example is using 154px.

![image](https://user-images.githubusercontent.com/1644143/114388548-bc7cff80-9b8b-11eb-83b8-880943072e65.png)

**Comment by aweell** [comment]
Hello @joaorbp,

Thanks for your feedback. We raised an issue related to the disparity in the component guidelines [here](https://github.com/dxc-technology/halstack-style-guide/issues/58#issue-856843060).

**Comment by mawelCaballero** [comment]
Hello @aweell , is this inconsistent has been fixed from component guidelines side? or is this sth to fix in angular cdk?

**Comment by aweell** [comment]
Hi @mawelCaballero,

Fixes to the [card documentation](https://developer.dxc.com/design/guidelines/components/card) have been added in release 3.1.1

* Content `padding`*
* `min-height` values
* image `max-width`

We need to be sure that we are compliant with the `min-height` and `max-width` specs. Regarding the font family, I've checked the latest releases and the component seems to use Open sans.

*In React CDK the padding properties have been divided into two:

* `imagePadding`
* `contentPadding`

We should follow the same implementation in both CDKs, component documentation will be updated in upcoming sprints to cover these properties.
